### PR TITLE
REL-3190: Remove Y and J-lo filters for Flamingos 2

### PR DESCRIPTION
**Pull request by cquiroz**
The PIT 2018A won't support these two filters

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Flamingos2.scala [hunk 1]
```diff
@@ -21,8 +21,8 @@ object Flamingos2 {
   private def filtersFor(d: Option[Flamingos2Disperser]): List[Flamingos2Filter] = d match {
     case Some(R1200HK) => List(JH, HK)
     case Some(R1200JH) => List(JH)
-    case Some(R3000)   => List(J_LOW, J, H, K_LONG, K_SHORT, K_BLUE, K_RED)
-    case _             => List(Y, J_LOW, J, H, K_LONG, K_SHORT, K_BLUE, K_RED)
+    case Some(R3000)   => List(K_LONG, K_SHORT, K_BLUE, K_RED)
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Flamingos2.scala]: Did you mean to also remove `J` and `H`?

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala [hunk 2]
```diff
@@ -134,7 +134,23 @@ case class LastStepConverter(semester: Semester) extends SemesterConverter {
...
+  val jLoFilter = "J-lo (1.122 um)"
+  val yFilter = "Y (1.020 um)"
+
+  val removeF2YJloFilters: TransformFunction = {
+    case p @ <flamingos2>{ns @ _*}</flamingos2> if (ns \ "filter").map(_.text).exists(f => f === jLoFilter || f === yFilter) =>
+      object yjLoFilterTransformer extends BasicTransformer {
+        override def transform(n: xml.Node): xml.NodeSeq = n match {
+          case <name>{name}</name>                                   => <name>{name.text.replaceAll(jLoFilter, "").replaceAll(yFilter, "")}</name>
+          case <filter>{filter}</filter> if filter.text == jLoFilter => xml.NodeSeq.Empty
+          case <filter>{filter}</filter> if filter.text == yFilter   => xml.NodeSeq.Empty
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: What happens if the proposal is left without any filters?  Do you lose the rest of the blueprint information?

Comment (cquiroz) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: No, the proposal and blueprint is still there but without a filter, you'd need to select a filter to submit

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Flamingos2.scala (hunk 1 above)
Comment (cquiroz) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Flamingos2.scala]: Oops

**Pull request merged**